### PR TITLE
New package: daemontools-0.76

### PR DESCRIPTION
**Pull request by bougyman**

### CODE REVIEW

**Comment by bougyman** [comment]
For a clearer view of this whole thing: http://phabricator.rubyists.com/D1

File: srcpkgs/daemontools/template [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'daemontools'
+pkgname=daemontools
+version=0.76
+revision=2
+wrksrc=admin/${pkgname}-${version}
```
Comment (xtraeme) [inline srcpkgs/daemontools/template]: `wrksrc` this way is wrong and while running the `clean` target it won't be fully removed.

Use this instead:

```
wrksrc=admin
build_wrksrc=${pkgname}-${version}
```

Comment (bougyman) [inline srcpkgs/daemontools/template]: Will do.

Comment (bougyman) [inline srcpkgs/daemontools/template]: Done

File: Manual.md [hunk 2]
```diff
@@ -544,6 +544,8 @@ be defined to set the allowed python versions to be built, i.e:
 
 - `waf` For packages that use the Python `waf` method with python2.
 
+- `slashpackage` For packages that use the /package heirarchy and package/compile to build.
```
Comment (xtraeme) [inline Manual.md]: `heirarchy` -> `hierarchy`

File: srcpkgs/daemontools/template [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Template file for 'daemontools'
+pkgname=daemontools
+version=0.76
+revision=1
+wrksrc=admin
```
Comment (xtraeme) [inline srcpkgs/daemontools/template]: `wrksrc` and `build_wrksrc` could be set in the environment for this `build_style`, this way:

```
void-packages/common/environment/build-style/slashpackage.sh
wrksrc=admin
build_wrksrc=${pkgname}-${version}
```

And then you should be able to remove those two vars in templates that set `build_style=slashpackage`.

Comment (bougyman) [inline srcpkgs/daemontools/template]: Packages aren't always going to be in 'admin'. There 'prog', and a bunch of other top-level categories. Is it ok to set build_wrksrc in the build_style script and then all that has to be filled in is the category (wrksrc) ?

**Comment by xtraeme** [comment]
Sure, I thought all them were using `admin`. In that case your change is ok.

**Comment by xtraeme** [comment]
Are not there manpages for those bins in daemontools?

File: Manual.md (hunk 2 above)
Comment (bougyman) [inline Manual.md]: Corrected

**Comment by bougyman** [comment]
Nope, no manpages. It has a README that just points to the website I have as 'homepage', didn't see a reason to include it.

**Pull request closed without merging**